lib/utils/retryability.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Returns the number of milliseconds to be used in an exponential backoff
+ * @param  base The base number of milliseconds to wait
+ * @param  attempt The number of times the operation has been attempted
+ * @param  max The maximum amount of milliseconds to wait
+ * @param  withJitter Set to true to add jitter (randomness) to the exponential backoff time that is returned
+ * @returns The number of milliseconds to be used in an exponential backoff
+ */
+export const getExponentialBackoffTimeInMS = (
```

Comment:
This is something I would also like to discuss with Bryan when he gets back because it could just be left over code from a first iteration that was not removed when he decided to use the default retry mechanism that the DynamoDB SDK offers.

Reply:
Last time I worked in that area, the `ThrottlingException` was not a retryable exception.

Reply:
When using the AWS SDK `Throttling Exception` is retried. https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Programming.Errors.html#Programming.Errors.RetryAndBackoff